src/components/AddingLayerForm/fields/EntryPointInput.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import { useForm } from 'react-final-form';
+import Alert from '@mui/material/Alert';
+import Box from '@mui/material/Box';
+import { getIn } from 'final-form';
+
+import { ScreenInputValues } from 'components/AddingLayerForm/types';
+import idMapToArray from 'helpers/idMapToArray';
+import isEmpty from 'helpers/isEmpty';
+
+import ScreenInput from 'components/AddingLayerForm/fields/ScreenInput';
+
+interface EntryPointInputProps {
+    name: string;
+}
+
+const EntryPointInput: React.FC<EntryPointInputProps> = ({ name }) => {
+    const form = useForm();
+    const { values } = form.getState();
+    const screens = getIn(values, `${name}.screens`) as ScreenInputValues;
```

Comment:
```suggestion
    const screens = useField<ScreenInputValues>(`${name}.screens`);
```

по идее при изменении скринов сюда будут прилетать новые значения и форсРендер не нужен

Reply:
Этот метод не вызывает ререндер. Если здесь использовать useField то каждый раз ререндерится весь список экранов. Не смог разобраться как правильней сделать :(

Reply:
вторым параметрам в useField можно передать конфиг и там кажется можно подписаться только на изменение value

Reply:
Да но только он считает за изменение все ниже идущие изменения и вызывает ререндер родителей. Например у меня через поле изменяется в реальном времени при вводе название экрана и стразу идет ререндер всех компонентов выше куда входит этот экран и в итоге доходит до списка эранов. У меня было сделано в начале серез useField но пределал на FormState не додумался мемоизацию попробовать использовать.